### ISSUE TITLE
Nexus Mod Manager IS DOWN

### ISSUE THREAD

**Issue by DaughertyUSMC** [Nexus-Mods/Nexus-Mod-Manager#311]
A few days ago, NMM simply stopped functioning in terms of logging in, downloading or installing, etc. I have searched all over the net trying to find out anything I can to repair this issue - No one is really talking about it, no solutions are offered - cannot find ANYTHING about this issue on the Nexus main site. Found a bunch of old posts on Steam AND Nexus about this happening years ago, but nothing current.
And before anyone suggests this or that, it is NOT my firewall, AV, Win Defender, or anything else related to my PC - I suspect it is the version I currently have (0.63.14) What I need is a reliable update link for the latest released version - tried several times with the linked executable (https://github.com/Nexus-Mods/Nexus-Mod-Manager/releases) someone posted for ver 0.65.4, here on GitHub - epic fail - won't load. It downloaded fine, but it fails to run/install. I suppose I could manually install mod files via the folder selection tool on NMM until something happens to re-enable the login server, but it would be nice to allow NMM to install some of the rather large mod file batches that I may not know how to properly place within my game directory.
Open to ideas

M.Daugherty
G/Sgt USMC

**Comment by Tektonic1** [comment]
Im having this exact same issue. I have no idea what to do, sorry.

**Comment by DaughertyUSMC** [comment]
Tektonic1, eventually someone may grace us with their knowledgeable presence here, and offer a solution - this is my "Glass half full" opinion - you don't wanna know what my "Glass half empty" response would be! (lol)
It's all good, amigo - we'll get it figured out, and if I do, I will post it here.

**Comment by rscottexe** [comment]
When you say "I have searched all over the net trying to find out anything" did you look at the one area where it might actually be worth looking -- The Nexus Mods news section (https://www.nexusmods.com/news/13481)? Or looking to see whether there's a new version of NMM on the download page visible on every NMM enabled game on Nexus Mods (https://www.nexusmods.com/mods/modmanager/)?

Either of those two (what I would call "quite obvious" locations) would have given you very quick answers to your problem!!!

**Comment by rscottexe** [comment]
The login servers for version 0.65.2 have worked for over 230,000 users thus far. If it's not working for you, it's very, very likely there's something blocking it on your end.

We no longer have any programmers who are able to work on NMM and thus do not provide support for NMM any more so unfortunately, you are on your own.

**Comment by DaughertyUSMC** [comment]
rscottexe - While your provided links Do in fact lead to the appropriate update info, and it IS appreciated, these were NOT easily visible - trust me - it's the first place I looked (nexus), so cool your jets friend. I found another way around the issue anyway. From your wording, am I to assume you are affiliated with Nexus? 

Issue closed already - reckon that answers my question.
For what it's worth, thanx!
